angular-frontend/src/app/app-config.service.ts

```diff
@@ -0,0 +1,44 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import {firstValueFrom} from "rxjs";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AppConfigService {
+
+  private appConfig: any;
```

Comment:
(Minor) Perhaps it'd be nice to have a type for this? 
```typescript
interface AppConfig {
  readonly scope?: string[];
  readonly clientId?: string;
  readonly authority: string;
}
```

Reply:
I like it